src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -423,6 +423,8 @@ def load_arguments(self, _):
 
     with self.argument_context('aks nodepool') as c:
         c.argument('cluster_name', help='The cluster name.')
+        # the following argument is declared for the wait command
+        c.argument('agent_pool_name', options_list=['--nodepool-name', '--agent-pool-name'], validator=validate_nodepool_name, help='The node pool name.')
```

Comment:
may unit test be added?

Reply:
Yes, added in commit [#7009a21](https://github.com/Azure/azure-cli-extensions/pull/5219/commits/7009a21f6ee542b69c9d7a117380def1b20625c5).